frontend/public/module/auth.js

```diff
@@ -36,8 +37,8 @@ export const authSvc = {
   email: () => loginStateItem(email),
 
   logout: (next) => {
-    setNext(next);
-    [userID, name, email].forEach(key => {
+    setNext(next || '/');
```

Comment:
This assumes that / is the basePath. You probably want `window.SERVER_FLAGS.basePath`